### DISCUSSION TITLE
YetiForce Documentation

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17104] [How to]
Hello,
Can someone tell me where I can get access to the YetiForce Documentation that was previously available on the YF page, for example, I had several shortcuts to pages with important information that I now need, however, now they are not there and on the new page to that I am redirected does not have the information I am looking for, for example https://yetiforce.com/en/knowledge-base/documentation/user-documentation/item/automation-of-notifications-in-tickets, I cannot find the information about [SLA] either Policy - Tickets to say something among many others...
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/3500b409-b25d-481a-bbba-4805e675f395)

**Comment by mariuszkrzaczkowski** [comment]
There is no access to the old documentation anymore.
We're currently creating new documentation and verifying and updating its contents.
If you need a specific article, please let us know in this thread, so we will consider it as a part of migrating the documentation.
